src/pocketmine/command/defaults/TimingsCommand.php

```diff
@@ -123,17 +135,12 @@ public function onCompletion(Server $server){
 							$server->getLogger()->logException($result);
 							return;
 						}
-						list(, $headers) = $result;
-						foreach($headers as $headerGroup){
-							if(isset($headerGroup["location"]) and preg_match('#^http://paste\\.ubuntu\\.com/([A-Za-z0-9+\/=]+)/#', trim($headerGroup["location"]), $match)){
-								$pasteId = $match[1];
-								break;
-							}
+						if(isset($result[0])){
+							$pasteId = json_decode($result[0], true)["id"];
```

Comment:
careful with your error handling here! `json_decode()` could return `false`, or the `id` offset might not exist.